packages/clerk-js/package.json

```diff
@@ -98,7 +98,7 @@
     "files": [
       {
         "path": "./dist/clerk.browser.js",
-        "maxSize": "58kB"
+        "maxSize": "66kB"
```

Comment:
Is this necessary  because of the createSlug helper? Got a few questions/points: 
- Is this something we definately need?
- Can it be down on the BE maybe?
- If we cannot remove it, we should move it within the /ui dir unless its used by /core. Moving it to ui will make it lazy-loaded. We should try to keep the entry module's size as low as possible/

Reply:
@nikosdouvlis I moved the createSlug helper into `ui/utils` and bundle size is back to norma